libraries/common/src/main/java/androidx/media3/common/text/VoiceSpan.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package androidx.media3.common.text;
+
+import static androidx.media3.common.util.Assertions.checkNotNull;
+
+import android.os.Bundle;
+import androidx.media3.common.util.UnstableApi;
+import androidx.media3.common.util.Util;
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+
+/**
+ * A span representing a speaker.
+ *
+ * <p>More information on <a href="https://www.w3.org/TR/webvtt1/#webvtt-cue-voice-span">
+ *   voice spans</a>.
+ */
+@UnstableApi
+public final class VoiceSpan {
+
+  /** The speaker name. */
```

Comment:
the webvtt spec uses the phrase "voice name", so I think we can use that here:

```
/** The voice name. */
public final String name;
```

Reply:
[340c14a](https://github.com/androidx/media/pull/1652/commits/340c14ad425062df753d1189114886130286d6e3)